sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_fl_scatter_gather_node.py

```diff
@@ -0,0 +1,131 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# pylint: disable=protected-access
+from typing import List, Dict, Optional, Union
+import importlib
+
+from azure.ai.ml.entities import PipelineJob
+from azure.ai.ml.entities._builders.fl_scatter_gather import FLScatterGather
+from azure.ai.ml.entities._assets.federated_learning_silo import FederatedLearningSilo
+from azure.ai.ml.entities import CommandComponent
+from azure.ai.ml._utils._experimental import experimental
+
+
+def _check_for_import(package_name):
+    try:
+        # pylint: disable=unused-import
+        importlib.import_module(package_name)
+    except ImportError:
+        raise ImportError(
+            "The DSL FL Node has an additional requirement above the rest of the "
+            + "AML SDK repo in that the mldesigner package is required. Please run `pip install mldesigner` "
+            + "and try again."
+        )
+
+
+@experimental
+def fl_scatter_gather(
```

Comment:
What is the expected SDK experience for users to import this API? as importing from a protected module (`from azure.ai.ml.dsl._fl_scatter_gather_node import fl_scatter_gather`) may look wired for users.